### PR TITLE
Money formats

### PR DESCRIPTION
**Pull request by montalvomiguelo**
### What are you trying to accomplish with this PR?
Add missing formats:

`amount_with_comma_separator`
`amount_with_apostrophe_separator`

Also fixes `amount_no_decimals_with_comma_separator ` which uses a dot as separator.

issue: #330

### CODE REVIEW

File: packages/slate-theme/test/currency.spec.js [hunk 1]
```diff
@@ -29,13 +29,23 @@ describe('currency.formatMoney', () => {
     expect(value).toBe('$100');
   });
 
-  test('Formats a number 1000001 to a string of "$10,000"', () => {
+  test('Formats a number 1000001 to a string of "$10.000"', () => {
```
Comment (t-kelly) [inline packages/slate-theme/test/currency.spec.js]: Not sure why you changed this? `no_decimals` meaning no cents value (e.g. `.00`), and the comma is now a `.`?

File: packages/slate-theme/test/currency.spec.js [hunk 2]
```diff
@@ -29,13 +29,23 @@ describe('currency.formatMoney', () => {
...
+    expect(value).toBe('$10.000');
   });
 
   test('Formats a number 1000001 to a string of "$10 000"', () => {
     var value = formatMoney(1000001, '${{amount_no_decimals_with_space_separator}}');
     expect(value).toBe('$10 000');
   });
+
+  test('Formats a number of 1000001 to a string of "$10.000,01"', () => {
+    var value = formatMoney(1000001, '${{amount_with_comma_separator}}');
```
Comment (t-kelly) [inline packages/slate-theme/test/currency.spec.js]: Same here. By separator, we're referring to the thousand seperator, where in the number `1,000,000` the `,` is the separator.

File: packages/slate-theme/test/currency.spec.js (hunk 1 above)
Comment (montalvomiguelo) [reply packages/slate-theme/test/currency.spec.js]: This is for `amount_no_decimals_with_comma_separator` money format which I have set in my store

I get `.` as separator by using the `money` filter

screen shot 2017-10-30 at 3 38 11 pm

Docs also are using a dot

{{ amount_no_decimals_with_comma_separator }} | ✓ | 1.135
-- | -- | --

https://help.shopify.com/manual/payments/currency-formatting

screen shot 2017-10-30 at 3 43 01 pm

File: packages/slate-theme/test/currency.spec.js (hunk 2 above)
Comment (montalvomiguelo) [reply packages/slate-theme/test/currency.spec.js]: By using `amount_with_comma_separator` the comma is used  to separate decimals 

screen shot 2017-10-30 at 3 55 03 pm

https://help.shopify.com/manual/payments/currency-formatting

{{ amount_with_comma_separator }} |   | 1.134,65
-- | -- | --

**Comment by t-kelly** [comment]
These scripts have been moved to https://github.com/Shopify/theme-scripts. Please open an issue to continue the discussion you started here. 

However please note that the use of `' '`, `','`, and `'.'` vary from region to region in their use for 1000 delimitations and decimal delimitations. The PR you opened here may make sense in your region, but doesn't make sense for a Canadian region.

**Pull request closed without merging**